files/ko/web/javascript/reference/global_objects/string/raw/index.html

```diff
@@ -0,0 +1,121 @@
+---
+title: String.raw()
+slug: Web/JavaScript/Reference/Global_Objects/String/raw
+tags:
+- ECMAScript 2015
+- JavaScript
+- Method
+- Reference
+- String
+- Polyfill
+translation_of: Web/JavaScript/Reference/Global_Objects/String/raw
+---
+<div>{{JSRef}}</div>
+
+<p>
+    <strong><code>String.raw()</code></strong> 메서드는 <a
+        href="/ko/docs/Web/JavaScript/Reference/Template_literals">템플릿리터럴</a>의 태그 함수입니다. 이는 Pyhon의 <code>r</code>접두사 또는
+    C#의 문자열 리터럴의
+    <code>@</code> 접두사와 유사합니다. (그러나 동일 하지는 않습니다. 이 문제에 관해서는 여기 <a
+        href="https://bugs.chromium.org/p/v8/issues/detail?id=5016">이슈</a>를 참조하십시오.) 이 메서드는 템플릿 리터럴의 윈시 문자열을 가져오는데
+    사용됩니다.
+    즉, 대체(예: <code>${foo}</code>)는 처리되지만 이스케이프(예: <code>\n</code>)는 처리되지 않습니다.
+</p>
+<p></p>
+
+<p> </p>
+
+<div>{{EmbedInteractiveExample("pages/js/string-raw.html")}}</div>
+
+
+
+<p> </p>
+
+<h2 id="구문">구문</h2>
+
+<pre class="brush: js">String.raw(callSite, ...substitutions)
+
+String.raw`templateString`
+</pre>
+
+<h3 id="매개변수">매개변수</h3>
+
+<dl>
+    <dt><code><var>callSite</var></code></dt>
+    <dd>정해진 형식의 템플릿 호출 개체 예:
+        <code>{ raw: ['foo', 'bar', 'baz'] }</code>.
+    </dd>
+    <dt><code><var>...substitutions</var></code></dt>
+    <dd><code><var>callSite</var></code> 사이사이에 대체값으로 포함됩니다. 예: <code>(['foo','baz'],'bar') => 'foobarbaz'</code></dd>
+    <dt><code><var>templateString</var></code></dt>
+    <dd>{{jsxref("template_literals", "템플릿 리터럴", "", 1)}}, 선택사항으로 대체를 포함 예:(<code>${...}</code>).</dd>
+</dl>
+
+<h3 id="반환값">반환값</h3>
+
+<p>주어진 템플릿 리터럴의 원시 문자열 형식.</p>
+
+<h3 id="예외">예외</h3>
+
+<dl>
+    <dt>{{jsxref("TypeError")}}</dt>
+    <dd><code>callSite</code> 인자의 객체의 포맷이 올바르지 않은 경우 {{jsxref("TypeError")}}예외가 발생(throw)한다.</dd>
+</dl>
+
```

Comment:
```
Description
In most cases, String.raw() is used with template literal. The first syntax mentioned above is only rarely used, because the JavaScript engine will call this with proper arguments for you, (just like with other tag functions).

String.raw() is the only built-in tag function of template literals. It works just like the default template function and performs concatenation. You can even re-implement it with normal JavaScript code.

```

예시 윗부분에 Description 부분이 있는데 누락됐습니다. 이것도 번역해주시면 좋을 것 같습니다! 어려우시다면 원문이라도 첨부해주세요.

Reply:
```suggestion
<h2 id="설명">설명</h2>

<p><code>String.raw()</code>의 경우 보통 템플릿 리터럴과 많이 사용하고, 첫번째 구문의 경우 잘 사용되지 않습니다.
왜냐하면 자바스크립트 엔진이  당신을 위해서 자동으로 적절한 인수로 호출해주기 때문입니다. ( 다른 태그 메서드들과 마찬가지로<a
    href="/ko/docs/Web/JavaScript/Reference/Template_literals#tagged_template_literals">태그 메서드</a>).</p>

<p><code>String.raw()</code>은 템플릿 리터럴의 유일한 내장 함수입니다. 기본 템플릿 기능과 동일하게 작동하며 연결을 수행합니다.  JavaScript 코드를 사용하여 다시 구현할 수도 있습니다.</p>
```